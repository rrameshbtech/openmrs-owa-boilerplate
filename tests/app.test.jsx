import React from "react";
import { shallow } from "enzyme";

import App from "../src/app.jsx";

describe("App Component", () => {
  it("should show welcome message", () => {
    const appComponent = shallow(<App />);

    expect(appComponent).toMatchSnapshot();
  });
});
