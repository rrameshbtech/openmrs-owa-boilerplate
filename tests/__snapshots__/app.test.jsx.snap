// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should show welcome message 1`] = `ShallowWrapper {}`;
